packages/main/src/ComboBox.ts

```diff
@@ -226,6 +226,17 @@ class ComboBox extends UI5Element {
 	@property()
 	value!: string;
 
+	/**
```

Comment:
missing since tag

Reply:
Latest version is 1.18.0. This feature will be part of next stable release which is 1.19.0.